scripts/bcl_thresholds.py

```diff
@@ -0,0 +1,51 @@
+"""Written by Isak Sylvin. isak.sylvin@scilifelab.se"""
+
+import sys
+
+class Thresholds():
+    def __init__(self, instrument, chemistry, paired):
+        self.valid_instruments = ['MiSeq', 'HiSeq', 'HiSeqX10']
+        self.valid_chemistry = ['MiSeq', 'HiSeq Rapid Flow Cell v1','HiSeq Rapid Flow Cell v2', 
+                             'TruSeq Rapid Flow Cell v2', 'TruSeq Rapid Flow Cell v3', 'HiSeq Flow Cell v4', 'HiSeqX10']
+        
+        self.Q30 = 0
+        self.exp_lane_clust = 0
+        
+        #Check only valid values are entered
+        if not instrument in self.valid_instruments or not chemistry in self.valid_chemistry:
+            sys.exit("Detected instrument, chemistry and/or read_setup are not present in bcl_thresholds.py")
+        else:
+            self.set_Q30(instrument, chemistry, paired)
+            self.set_exp_lane_clust(instrument, chemistry, paired)
+    
+    #Q30 values here go for max read length
+    #If read length should be a factor, values need to be upd.
```

Comment:
insert comment about the fact that threshold reflect document 1262 and 1244 in AM-System (therefore integrate also read -lenght checks)